lib/algolia/index.rb

```diff
@@ -1295,6 +1295,11 @@ def exists
       return true
     end
 
+    #
+    # Aliases the exists method
+    #
+    alias exists? exists
```

Comment:
It's valid, I just have a preference for `alias :exists? :exists` as it shows a clear reference to a method. But this is nitpicking.

Reply:
I thought that you had to use the `:` only with `alias_method`. Would you favor `alias_method`?

Reply:
There are slight differences between the two of them but in this case both works. No the `:` works as well for `alias`